src/NHapi.Base/Model/ISegment.cs

```diff
@@ -27,33 +27,36 @@
 
 namespace NHapi.Base.Model
 {
-   /// <summary> Represents an HL7 message segment, which is a unit of data that contains multiple fields.</summary>
-   /// <author>  Bryan Tripp (bryan_tripp@sourceforge.net)
-   /// </author>
+   /// <summary>
+   /// Represents an HL7 message segment, which is a unit of data that contains multiple fields.
+   /// </summary>
+   /// <author>Bryan Tripp (bryan_tripp@sourceforge.net)</author>
    public interface ISegment : IStructure
 	{
-		/// <summary> Returns the array of Fields at the specified index.  The array will be of length 1 for
+		/// <summary>
+		/// Returns the array of Fields at the specified index.  The array will be of length 1 for
 		/// non-repeating fields, and >1 for repeating fields.  Fields are numbered from 1.
 		/// </summary>
 		/// <throws>  HL7Exception if field index is out of range. </throws>

```

Comment:
```csharp
/// <exception cref="HL7Exception">if field index is out of range.</exception>
```

though I know this is not really part of fixing the build warnings so im not expecting you to change this.